src/target/riscv/riscv.c

```diff
@@ -4040,8 +4040,7 @@ COMMAND_HANDLER(handle_info)
 	return 0;
 }
 
-COMMAND_HANDLER(riscv_exec_progbuf)
-{
+COMMAND_HANDLER(riscv_exec_progbuf) {
```

Comment:
At a glance, I don't see any COMMAND_HANDLER() examples where { is on the same line.